gradle.properties

```diff
@@ -56,10 +57,10 @@ marathonClientVersion=0.6.2
 micrometerVersion=1.2.0
 mockitoVersion=3.3.3
 jacocoVersion=0.8.5
-nettyVersion=4.1.50.Final
-nettyBoringSSLVersion=2.0.30.Final
-protobufGradlePlugin=0.8.9
-protobufProtocVersion=3.13.0
+nettyVersion=4.1.59.Final
```

Comment:
@derekm let's update netty and other relevant libraries to their latest stable versions. There is 4.1.65 version available which is considered latest stable and recommended by Netty.

Reply:
@medvedevigorek -- Upgrading to latest seems to break the HDFS connector. I'll try incrementing through the versions and see where it breaks or if we're already at the latest working version.